notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    numpyro.set_host_device_count(4)
</pre><p>Maybe it is not necessary to use 4 devices and 4 chains?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Not sure why the progress_bars in your MCMC run is so long. Maybe it depends on the screen resolution...</p>